squash_wip.go

```diff
@@ -186,7 +186,7 @@ func isManualCommit(line string, configuration Configuration) bool {
 }
 
 func isWipCommit(line string, configuration Configuration) bool {
-	return strings.HasSuffix(line, configuration.WipCommitMessage)
+	return strings.Contains(line, configuration.WipCommitMessage)
```

Comment:
why contains and not hasPrefix similarly to the change above?

Reply:
As you can see, the function used has suffix before because there is a abbreviated commit hash in Front on the commit message. I am not sure if this hash is needed of if we could change the code that we could just get the message here.